lib/msal-browser/src/index.ts

```diff
@@ -27,9 +28,6 @@ export {
     AuthenticationScheme,
     // Account
     AccountInfo,
-    // Request
-    AuthorizationUrlRequest,
```

Comment:
This would be a breaking change. You can do thins: `export { CommonAuthorizationCodeRequest as AuthorizationUrlRequest }`

Reply:
Prithvi and I talked about this. It's technically a breaking change anyway because we're changing the underlying type but seeing as how people shouldn't be using this we wanted to remove it, if we need to bump the version accordingly, we can.

Reply:
It looks like the exported names in common are also changing. We need to guard against breaking apps that pick up teh new version of common but not the new version of browser.

Reply:
If the main reason for this is naming, then I would say leverage the `as` renaming for imports and exports, e.g `import { AuthorizationCodeRequest as CommonAuthorizationCodeRequest } from '@azure/msal-browser';`

Reply:
Otherwise if we need to make this a breaking change in common, we should do that.

Reply:
Not just naming, we're changing required vs. optional params on each of the types to ensure that requests passed around internally have all the required params without making them required on the API surface. It's a breaking change either way.

Reply:
It certainly seems like common should be a breaking change. Given `AuthorizationUrlRequest` shouldn't be used directly, I'm less concern about that changing from the browser export.

Reply:
For the browser request types that built off of `AuthorizationUrlRequest` (e.g. popuprequest), are we confident that the params that were being used haven't changed in a breaking way? e.g. making a param in popuprequest that was being used being changed from an optional to non-optional property.

Reply:
Yes because all those request types use the `Partial` utility type to make everything optional and then explicitly declares it's required types, if any.

Reply:
Now being exported directly from browser